src/main/resources/com/puppycrawl/tools/checkstyle/checks/metrics/messages_tr.properties

```diff
@@ -11,5 +11,5 @@ cyclomaticComplexity = Cyclomatic Complexity değeri {0,number,integer} {0,numbe
 ncss.class  = Bu sınıfın açıklama olmayan kaynak kod satırı (NCSS) sayısı  {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
 ncss.file   = Bu dosyanın açıklama olmayan kaynak kod satırı (NCSS) sayısı  {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
 ncss.method = Bu metodun açıklama olmayan kaynak kod satırı (NCSS) sayısı  {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-
+ncss.record=Açıklama olmadan bu kaydın kaynak kodu satırı (NCSS) sayısı  {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
 npathComplexity = NPath Complexity değeri {0,number,integer} (maksimum izin verilen değer {1,number,integer}). (NPath Complexity: Bir metodun mümkün olan tüm gerçeklenme yollarının sayısı)
```

Comment:
More likely it is "Bu `kaydın` açıklama olmayan kaynak kod satırı (NCSS) sayısı"